help/about-mp.txt

```diff
@@ -0,0 +1,22 @@
+ <- @@index|<<Back to Index>>@@
```

Comment:
the one from manaplus also includes copyright for tmw and some links do now work; I want that page to be only about manaplus, not manaplus+tmw+evol